dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/EPersonRestRepository.java

```diff
@@ -179,6 +187,59 @@ public EPersonRest findByEmail(@Parameter(value = "email", required = true) Stri
         return converter.fromModel(eperson);
     }
 
+    @Override
+    @PreAuthorize("hasAuthority('ADMIN')")
+    public void patch(Context context, HttpServletRequest request, String apiCategory, String model, UUID uuid,
+                      Patch patch)
+            throws UnprocessableEntityException, PatchBadRequestException, AuthorizeException,
+            ResourceNotFoundException {
+
+        try {
+            EPerson eperson = es.find(context, uuid);
+            if (eperson == null) {
+                throw new ResourceNotFoundException(apiCategory + "." + model + " with id: " + uuid + " not found");
+            }
+            List<Operation> operations = patch.getOperations();
+            EPersonRest ePersonRest = findOne(context, uuid);
+            EPersonRest patchedModel = (EPersonRest) epersonPatch.patch(ePersonRest, operations);
+            updatePatchedValues(context, patchedModel, eperson);
+
+        } catch (SQLException e) {
+            throw new RuntimeException(e.getMessage(), e);
+        }
+    }
+
+    /**
+     * Applies changes in the rest model.
+     * @param context
+     * @param ePersonRest the updated eperson rest
+     * @param ePerson the eperson content object
+     * @throws SQLException
+     * @throws AuthorizeException
+     */
+    private void updatePatchedValues(Context context, EPersonRest ePersonRest, EPerson ePerson)
+            throws SQLException, AuthorizeException {
+
+        if (ePersonRest.getPassword() != null) {
+            es.setPassword(ePerson, ePersonRest.getPassword());
+        }
+
+        if (ePersonRest.isRequireCertificate() != ePerson.getRequireCertificate()) {
+            ePerson.setRequireCertificate(ePersonRest.isRequireCertificate());
+        }
+        if (ePersonRest.isCanLogIn() != ePerson.canLogIn()) {
+            ePerson.setCanLogIn(ePersonRest.isCanLogIn());
+        }
+        if (ePersonRest.getNetid() != null) {
+            if (!ePersonRest.getNetid().equals(ePerson.getName())) {
```

Comment:
this looks odd

Reply:
I think this was my attempt to avoid NPE.  After checking, I decided to replace with `!Objects.equals(ePersonRest.getNetid(), ePerson.getNetid())`. 

Perhaps a bit less odd.